libr/bin/demangle.c

```diff
@@ -295,6 +295,62 @@ R_API char *r_bin_demangle_objc(RBinFile *binfile, const char *sym) {
 	return ret;
 }
 
+static inline bool rust_replace_mangled_seq (const char **in, char **out, const char *seq, char value) {
```

Comment:
as long as is static use a shorter name. drop the inline because this function is called a lot of times in a block which will produce a huge bottleneck in executed if inlined